### ISSUE TITLE
webapp: Problem trying to update container image via CLI

### ISSUE THREAD

**Issue by ewassef** [Azure/azure-cli#12996]
## Describe the bug

**Command Name**
`az webapp config container set`

**Errors:**
```
'NoneType' object has no attribute 'reserved'
Traceback (most recent call last):
Temp\pip-install-wl9gly17\knack\knack\cli.py, ln 206, in invoke
azure\cli\core\commands\__init__.py, ln 566, in execute
azure\cli\core\commands\__init__.py, ln 624, in _run_jobs_serially
azure\cli\core\commands\__init__.py, ln 617, in _run_job
Local\Temp\pip-install-wl9gly17\six\six.py, ln 693, in reraise
azure\cli\core\commands\__init__.py, ln 594, in _run_job
azure\cli\core\commands\__init__.py, ln 303, in __call__
azure-cli-core\azure\cli\core\__init__.py, ln 470, in default_command_handler
azure\cli\command_modules\appservice\custom.py, ln 855, in update_container_settings
azure\cli\command_modules\appservice\custom.py, ln 612, in _add_fx_version
AttributeError: 'NoneType' object has no attribute 'reserved'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az webapp config container set --docker-custom-image-name {} --name {} --resource-group {} --docker-registry-server-password {} --docker-registry-server-url {} --docker-registry-server-user {} --subscription {}`

## Environment Summary
```
Windows-10-10.0.18362-SP0
Python 3.6.6
Shell: cmd.exe

azure-cli 2.0.67 *

```

**Comment by yonzhan** [comment]
webapp

**Comment by JeffinBao** [comment]
I am facing the exact same issue, any updates?

**Comment by panchagnula** [comment]
@calvinsID does your latest change fix this None error?

**Comment by calvinsID** [comment]
Fixed by #14633
